packages/fxa-auth-server/lib/payments/paypal/helper.ts

```diff
@@ -485,16 +485,23 @@ export class PayPalHelper {
       paymentAttempt
     );
 
-    const countryCode: string =
-      invoice.customer_shipping?.address?.country ??
-      this.currencyHelper.currencyToCountryMap[
-        invoice.currency.toUpperCase()
-      ][0];
+    let countryCode: string | undefined =
+      invoice.customer_shipping?.address?.country ?? undefined;
 
     if (!countryCode) {
-      throw error.internalValidationError('processInvoice', {
-        message: 'Invalid country code',
-      });
+      const validCountries = this.currencyHelper.currencyToCountryMap.get(
+        invoice.currency.toUpperCase()
+      );
+
+      if (validCountries && validCountries.length > 0) {
+        countryCode = validCountries[0];
+      } else {
+        this.log.error('processInvoice', {
```

Comment:
nit. Could `processInvoice` be updated to something unique, like `processInvoice.countryCode`. It'll make it easier to find this specific log in BQ.